doc/user-settings.md

```diff
@@ -204,9 +204,20 @@ used by default by overriding `buildProgressColour` and `successColour`:
 ```haskell
 -- | Set colour for build progress messages (e.g. executing a build command).
 buildProgressColour :: BuildProgressColour
-buildProgressColour = BuildProgressColour (Dull, Magenta)
+buildProgressColour = BuildProgressColour (Dull Magenta)
```

Comment:
Should this be `mkBuildProgressColour` instead? Same inconsistency below.

Reply:
Oh, yes you are right. I was very confused how that compiled for a moment, but then realized this was the documentation.

Reply:
@patrickdoc Are you planning to fix this or shall I just merge and fix the docs?

Reply:
Sorry for the hold up! I thought you were going to be making edits and could just fold it in. I've fixed it now.

Reply:
@patrickdoc Thanks! No problem about the delay, I was waiting for some feedback from other Hadrian developers anyway. Let me ping them again :)